### PR TITLE
 feature Reject/Approve Trip Request 182201726

### PR DESCRIPTION
**Pull request by alaintresor**
#### What does this PR do?
Approve /Reject Trip Request
#### Description of Task to be completed?

- Approve trip request by manager
- Reject trip request by manager
- View all trip request

#### How should this be manually tested?
After running the app with `npm run start:dev` and **login as manager** and after using this endpoint `/api/v1/trip-request/approve/[TRIP-REQUEST ID]` on a PUT request to Approve trip request or using this endpoint `/api/v1/trip-request/reject/[TRIP-REQUEST ID]` on a PUT request to Reject a trip request.
`GET/api/v1/trip-request/` to view all trip requests
OR
after starting a server, you can open any web browser and put this URL'localhost:[YOUR PORT]/api/v1/docs' to test it with swagger

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/182201726

### CODE REVIEW

**Review by uwambajeddy — DISMISSED** [review]
LGTM

File: test/tripRequest.test.js [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+        api
+        .post('/api/v1/user/auth/signup')
+          .send(user)
+        .end((err, res) => {
+          const { token } = res.body;
+            api
+            .put('/api/v1/trip-request/approve/1')
+             .set('Authorization',`Bearer ${token}`)
+             .end((err,res)=>{
+                    expect(res.status).to.equal(200);
```
Comment (joshwambere) [inline test/tripRequest.test.js]: you might want tto also test the right message is being deliverd

**Comment by joshwambere** [comment]
and solve some conflicts on this PR

**Review by joshwambere — DISMISSED** [review]
LGTM

**Review by uwambajeddy — DISMISSED** [review]

**Review by Placide123 — DISMISSED** [review]

**Review by mwisemarierose — DISMISSED** [review]

**Review by Patricknshimiyimana — DISMISSED** [review]

File: src/controllers/tripRequest.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
+
```
Comment (joshwambere) [inline src/controllers/tripRequest.js]: why do you have two trip request controllers?

**Review by joshwambere — CHANGES_REQUESTED** [review]

**Pull request closed without merging**